app/src/main/java/com/lab/skuld/ui/LiveArray.kt

```diff
@@ -66,5 +68,5 @@ fun <T, F> rememberLiveArray(
         }
     }
 
-    return documents
+    return remember { derivedStateOf { documents } }
```

Comment:
Hope this won't brake the import feature(I'll check and fix it if it does, just saying)